src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -30,4 +33,83 @@ public boolean connect(String database, String user, String password) {
             return false;
         }
     }
+
+    public void close() throws SQLException {
+        if (connection != null) {
+            connection.close();
+        }
+    }
+
+    public String[] getTableNames() throws SQLException {
+        try (ResultSet rs = connection.getMetaData().getTables(
+                null, "public", "%", new String[]{"TABLE"})) {
+            int tablesCount = 0;
+            if (rs.last()) {
+                tablesCount = rs.getRow();
+                rs.beforeFirst();
+            }
+            String[] tableNames = new String[tablesCount];
+            int index = 0;
+            while (rs.next()) {
+                tableNames[index++] = rs.getString("table_name");
+            }
+            return tableNames;
+        }
+    }
+
+    public DataSet[] getTableData(String tableName) throws SQLException {
+        if (!isTableExists(tableName)) {
+            return new DataSet[0];
+        }
+        int tableSize = TableSize(tableName);
+        if (tableSize == 0) {
+            return new DataSet[0];
+        }
+        DataSet[] dataSets = new DataSet[tableSize];
+        int numberOfColumns = NumberOfColumns(tableName);
+        fillDataSets(tableName, dataSets, numberOfColumns);
+        return dataSets;
+    }
+
+    private boolean isTableExists(String tableName) throws SQLException {
+        for (String name : getTableNames()) {
+            if (name.equals(tableName)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private void fillDataSets(String tableName, DataSet[] result, int numberOfColumns) throws SQLException {
+        try (Statement stmt = connection.createStatement();
+             ResultSet rs = stmt.executeQuery("SELECT * FROM public." + tableName)) {
+            int index = 0;
+            while (rs.next()) {
+                DataSet dataSet = new DataSet(numberOfColumns);
+                result[index++] = dataSet;
+                for (int i = 0; i < numberOfColumns; i++) {
+                    dataSet.insertValue(i, rs.getString(i + 1));
+                }
+            }
+        }
+    }
+
+    private int NumberOfColumns(String tableName) throws SQLException {
```

Comment:
Method names should begin with a lowercase letter

Reply:
tnx